README.md

```diff
@@ -176,16 +176,20 @@ When you want to use prisma migrate dev without seeding, you can pass the --skip
 
 Create a `.env` file in project root. If running with docker an additional `.env.dev` file is needed. Refer to below example for env variables:
 
-```
+```bash
 PORT=9000
 DATABASE_URL="postgresql://postgres:postgres@postgres:5432/claim"
 ACCESS_SECRET=dPEBknfdAcx5bir34KnX2mATWZnvM4xF
```

Comment:
ugh remove ACCESS_SECRET i know you did put put it but remove it please

Reply:
Done!